webui/react/src/components/UPlotChart.tsx

```diff
@@ -11,6 +11,11 @@ export interface Options extends Omit<uPlot.Options, 'width'> {
   width?: number;
 }
 
+interface SerieMaxMin {
```

Comment:
Not sure about calling it a Serie, though if we already use that term elsewhere then it's alright. Not sure about a better option either. `SingleSeriesMaxMin`?

Reply:
Oh good catch, that should be ScaleMaxMin, and yes, adding a Single works :)

Will change this